cmd/drone-server/inject_store.go

```diff
@@ -55,6 +56,7 @@ var storeSet = wire.NewSet(
 	global.New,
 	step.New,
 	template.New,
+	provideRedisClient,
```

Comment:
Is this the right place for this? We on load stores here. Feels like it would belong more in inject_service?

Reply:
I'm not sure either. Brad suggested to add it to inject_store.go. But in reality it's in its own category. Maybe it would be the best to create a new file.

Reply:
Yeah I'd agree with you there.

Reply:
changed... moved to a new file inject_external.go